### PR TITLE
Revise metrics definition

### LINKED ISSUE #11514: R squared value should be better when closer to 1
&gt; R Squared is the variation percentage in the predicted 
&gt; values explained by your model. It's a value between 0
&gt; and 1, and the closer the value is to 0, the better the 
&gt; model is.

I'm used to the R Squared value being closer to 1 when the result is better http://blog.minitab.com/blog/adventures-in-statistics-2/regression-analysis-how-do-i-interpret-r-squared-and-assess-the-goodness-of-fit

If the stated paragraph is correct for ML.NET, it may help to explain why this R squared value is different from the others. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0888bc80-798f-ebdd-459a-7543be1efdad
* Version Independent ID: c24ded82-d56c-e8c7-fd6a-e39929227177
* Content: [Use ML.NET in a movie recommendation scenario](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/movie-recommmendation#feedback)
* Content Source: [docs/machine-learning/tutorials/movie-recommmendation.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/movie-recommmendation.md)
* Product: **dotnet-ml**
* GitHub Login: @briacht
* Microsoft Alias: **johalex**

### PR DESCRIPTION
**Pull request by JRAlexander**
Revise metrics definition and added link to improving your model.
Fixes #11514

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/movie-recommmendation?branch=pr-en-us-11667)

### CODE REVIEW

**Review by luisquintanilla — APPROVED** [review]
LGTM. Thanks @JRAlexander

File: docs/machine-learning/tutorials/movie-recommmendation.md [hunk 1]
```diff
@@ -300,9 +300,11 @@ RSquared: 0.412556298844873
 
 In this output, there are 20 iterations. In each iteration, the measure of error decreases and converges closer and closer to 0.
 
-The `root of mean squared error` (RMS or RMSE) is frequently used to measure the differences between values predicted by a model and the values observed in a test dataset. Technically it's the square root of the average of the squares of the errors. You want your RMSE score to be as close to 1 as possible.
+The `root of mean squared error` (RMS or RMSE) is frequently used to measure the differences between values predicted by a model and the values observed in a test dataset. Technically it's the square root of the average of the squares of the errors. The lower it is, the better the model is.
```
Comment (natke) [inline docs/machine-learning/tutorials/movie-recommmendation.md]: I would remove "frequently" here

**Review by natke — APPROVED** [review]
One minor comment but otherwise all good.

File: docs/machine-learning/tutorials/movie-recommmendation.md (hunk 1 above)
Comment (JRAlexander) [reply docs/machine-learning/tutorials/movie-recommmendation.md]: Good catch. Thanks, @natke!

**Comment by JRAlexander** [comment]
Thanks, @luisquintanilla!

**Pull request merged**